### ISSUE TITLE
UID y PASS

### ISSUE THREAD

**Issue by enrique131** [pymedusa/Medusa#11342]
**Describe the bug**

There is a page with instructions on the wiki that is outdated:
https://github.com/pymedusa/Medusa/wiki/Configure-Providers-with-captcha-protection

In order to get that information via Google chrome steps are:
with the site you want to get the information open, go to the 3 dots (top right of chrome) -- > more tools --> dev tools --> application --> cookies (left menu)

hope this helps someone to avoid losing as much time as i did.

**Comment by jaxjexjox** [comment]
Ok yes something has broken with me too.  My provider, whome I've been with a very long time and Medusa has worked great with, is now spitting out problems.

I've just updated Medusa, to be sure to the latest docker build


 Medusa Info:	Branch: master
Commit: e98773f869de7c132b45fa8c76fa66cfc64d1e7b
Version: 1.0.16
Database: 44.19
 Python Version:	3.10.8 (main, Oct 13 2022, 23:33:11) [GCC 10.3.1 20211027]
 SSL Version:	OpenSSL 1.1.1q 5 Jul 2022
 OS:	Linux-5.15.0-72-generic-x86_64-with
 Locale:	en_US.UTF-8
 Timezone:	AEST





2023-06-09 16:24:49 INFO	FORCEDSEARCHQUEUE-MANUAL-2710 :: XXXXXXXXXXXXXX:: [e98773f] Performing season pack search for It's ZZZZZZZZZZZZZZ
2023-06-09 16:24:49 WARNING	FORCEDSEARCHQUEUE-MANUAL-2710 :: XXXXXXXXXXXXXX:: [e98773f] Please configure the required cookies for this provider. Check your provider settings
2023-06-09 16:24:49 INFO	FORCEDSEARCHQUEUE-MANUAL-2710 :: XXXXXXXXXXXXXX:: [e98773f] Unknown exception in url https://XXXXXXXXXXXXXX.me Error: Detected a Cloudflare version 2 Captcha challenge, This feature is not available in the opensource (free) version.
2023-06-09 16:24:49 INFO	FORCEDSEARCHQUEUE-MANUAL-2710 :: XXXXXXXXXXXXXX:: [e98773f] Performing episode search for ZZZZZZZZZZZZZZ




Double checked my cookie stuff is ok.

**Comment by jaxjexjox** [comment]
@enrique131  Actually I've just re-read your post.

So I do have the correct cookies and have done for a very long time.    Yet mine is spitting out errors about them not being configured.

And yes, Chrome seems to have moved the spot to pull the cookie info, but I signed in on Edge, noticed that the UID and pass are indeed the same as was configured, in my Medusa, long, long ago.

**Comment by enrique131** [comment]
Yes, now even with the right UID and pass still doesn't seem to work.

**Comment by rknightion** [comment]
Same here for IPT. I guess either they or CloudFlare have changed something? Given Prowlarr is also suffering the same issue I suspect its a more wider change than just pymedusa!

**Comment by enrique131** [comment]
> Same here for IPT. I guess either they or CloudFlare have changed something? Given Prowlarr is also suffering the same issue I suspect its a more wider change than just pymedusa!

Oh... good to know. i will wait then to see if the internet magicians find a solution or alternative. Thanks

**Comment by jaxjexjox** [comment]
So presumably someone else is tracking this on another thread somewhere, for another product, since it's not Medusas fault (thank you devs!)

Anyone know where this might be discussed?  Prowlarr issues feedback forum?

**Comment by nstephenh** [comment]
IPT started working for me after I saved again with the URL manually set. Unsure what exactly happened, but it could be a difference in the .me url vs the .com url. 

Mostly came here to suggest updating the documentation.

**Comment by Wabad00** [comment]
> IPT started working for me after I saved again with the URL manually set. Unsure what exactly happened, but it could be a difference in the .me url vs the .com url.
> 
> Mostly came here to suggest updating the documentation.

I came here to say the same thing.  Custom URL using the .com made everything work again

**Comment by jaxjexjox** [comment]
> > IPT started working for me after I saved again with the URL manually set. Unsure what exactly happened, but it could be a difference in the .me url vs the .com url.
> > Mostly came here to suggest updating the documentation.
> 
> I came here to say the same thing. Custom URL using the .com made everything work again


Sorry, I'm a tad confused, what would the custom URL actually be?
I tried using just www.ipt.com  - no dice with the test button.

**Comment by Wabad00** [comment]
https://www.iptorrent.com/

**Comment by skeelkat** [comment]
> https://www.iptorrent.com/

This works for me as well.

**Comment by jaxjexjox** [comment]
torrentleech has a similar bug now, but no custom URL :/


 Unknown exception in url https://www.torrentleech.org/torrents/browse/list/categories/26,27,32,34,35,44/orderby/added/order/desc/ Error: Unable to find Cloudflare cookies. Does the site actually have Cloudflare IUAM (I'm Under Attack Mode) enabled?

**Comment by madtempest** [comment]
> https://www.iptorrent.com/

I think you are missing the s in this URL.

**Comment by JarnoStoffer** [comment]
Having this issue too - does Medusa not support setting the "cf_clearance" cookie?

**Comment by medariox** [comment]
> Having this issue too - does Medusa not support setting the "cf_clearance" cookie?

It does. You can do it in the provider settings, see this guide: https://github.com/pymedusa/Medusa/wiki/Configure-Providers-with-captcha-protection

**Comment by medariox** [comment]
com domain is now default. Configure your cookies again if you have to, see: https://github.com/pymedusa/Medusa/wiki/Configure-Providers-with-captcha-protection
